index.html

```diff
@@ -73,6 +73,17 @@ <h3>Photography</h3>
         </div>
       </div>
     </main>
+
+    <div class="images__four">
+      <img class="images__four--mobile" src="build/img/mobile/image-gallery-milkbottles.webp" alt="image milk bottles">
+      <img class="images__four--desktop" src="build/img/desktop/image-gallery-milkbottles.webp" alt="image milk bottles">
+      <img class="images__four--mobile" src="build/img/mobile/image-gallery-orange.webp" alt="">
+      <img class="images__four--desktop" src="build/img/desktop/image-gallery-orange.webp" alt="">
+      <img class="images__four--mobile" src="build/img/mobile/image-gallery-cone.webp" alt="">
+      <img class="images__four--desktop"src="build/img/desktop/image-gallery-cone.webp" alt="">
```

Comment:
you should add some description to the alt attribute, also move that to a picture tag instead